docs/advanced/s3.md

```diff
@@ -0,0 +1,248 @@
+        uses: actions/checkout@v4
+
+      # temporary credentials via OIDC
+      - name: Log in to AWS
+        uses: aws-actions/configure-aws-credentials@v4
+        id: aws
+        with:
+          role-to-assume: arn:aws:iam::123456789012:role/github-poweruser
+          aws-region: eu-central-1
+
+      - name: Set up pixi
+        # AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are set by aws-actions/configure-aws-credentials
+        uses: prefix-dev/setup-pixi@v0.8.2
+        with:
+          auth-aws-access-key-id: ${{ steps.aws.outputs.aws-access-key-id }}
+          auth-aws-secret-access-key: ${{ steps.aws.outputs.aws-secret-access-key }}
+          auth-aws-session-token: ${{ steps.aws.outputs.aws-session-token }}
+          auth-host: s3://my-s3-bucket
+```
+
+## Public S3 buckets
+
+You can also use public S3 buckets without authentication.
+For this, you need to configure your S3 bucket in such a way that it allows public access.
+On AWS, for example, you need the `GetObject` and `ListBucket` permissions for this.
+Here is an example for AWS S3:
+
+```json title="Bucket policy"
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PublicReadGetObject",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": "s3:GetObject",
+            "Resource": "arn:aws:s3:::my-public-bucket/*"
+        },
+        {
+            "Sid": "PublicReadListBucket",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": "s3:ListBucket",
+            "Resource": "arn:aws:s3:::my-public-bucket"
+        }
+    ]
+}
+```
+
+## S3-compatible storage
+
+Many other cloud providers offer S3-compatible storage APIs.
+You can use them with pixi by specifying the `s3-options` in your manifest file.
+
+### MinIO
+
+```toml title="pixi.toml"
+[project.s3-options]
+endpoint-url = "https://minio.example.com"
+region = "us-east-1"
+force-path-style = true
+```
+
+### Cloudflare R2
+
+```toml title="pixi.toml"
+endpoint-url = "https://<account-id>.eu.r2.cloudflarestorage.com"
+region = "WEUR"
+force-path-style = false
+```
+
+### Wasabi
+
+```toml title="pixi.toml"
+endpoint-url = "https://s3.de-1.wasabisys.com"
+region = "de-1"
+force-path-style = false
+```
+
+### Backblaze B2
+
+```toml title="pixi.toml"
+endpoint-url = "https://s3.us-west-004.backblazeb2.com"
+region = "us-west-004"
+force-path-style = true
+```
+
+### Google Cloud Storage
+
+Note pixi also supports `gcs://` URLs.
+
+```toml title="pixi.toml"
+endpoint-url = "https://storage.googleapis.com"
+region = "us-east-1"
+force-path-style = false
+```
+
+### Hetzner Object Storage
+
+```toml title="pixi.toml"
+endpoint-url = "https://fsn1.your-objectstorage.com"
+region = "US"
+force-path-style = false
+```
+
+## Uploading to S3
+
+You can upload to S3 using `rattler-build upload s3`.
+For more information, see [rattler-build's documentation](https://rattler.build/latest/authentication_and_upload/#s3).
```

Comment:
https://github.com/prefix-dev/rattler-build/pull/1299#discussion_r1937350814

Reply:
Let's remove it for now, to avoid confusion

Reply:
it's commented out right now so nobody should see this. is this okay as well?